### PR TITLE
updated button appearance in start activity

### PR DESCRIPTION
**Pull request by mahikaw**
Added a new drawable with blank button and then added the text onto the buttons separately.
Now the appearance is uniform. Addresses #258.
@chhavip @aanchal07 Please review and merge :)

### CODE REVIEW

**Comment by aanchal07** [comment]
Good work. 
Can you also add the same for "Continue" and "Back" button in other activities to maintain the consistency across the whole application? 
@chhavip What do you say?

**Comment by mahikaw** [comment]
Yeah sure @aanchal07 , I'll do that right away. :)

**Comment by chhavip** [comment]
@aanchal07 good suggestions, can be included in this PR itself if @mahikaw iswilling.

**Comment by mahikaw** [comment]
@chhavip @aanchal07 I have made the changes for all buttons throughout, please review and merge. :)

File: PowerUp/app/src/main/res/layout-land/activity_select_features.xml [hunk 1]
```diff
@@ -522,6 +522,6 @@
         android:layout_above="@+id/power_bar"
         android:layout_alignParentEnd="true"
         android:layout_alignParentRight="true"
-        android:background="@drawable/endingscreen6" />
+        android:background="@drawable/buttons" />
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout-land/activity_select_features.xml]: Give drawable a better name. somethign like: brown_button_background

File: PowerUp/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -29,22 +29,28 @@
...
 
-    <ImageButton
+    <Button
         android:id="@+id/aboutButtonMain"
         android:layout_width="@dimen/about_button_width"
         android:layout_height="@dimen/about_button_height"
-        android:background="@drawable/aboutbutton" />
+        android:background="@drawable/buttons"
+        android:text="@string/About"
+        android:textSize="18dp"/>
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/activity_main.xml]: Do not hardcode, use dimens.xml, at all other places in the code too.

File: PowerUp/app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -29,22 +29,28 @@
...
 
-    <ImageButton
+    <Button
         android:id="@+id/aboutButtonMain"
         android:layout_width="@dimen/about_button_width"
         android:layout_height="@dimen/about_button_height"
-        android:background="@drawable/aboutbutton" />
+        android:background="@drawable/buttons"
+        android:text="@string/About"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/activity_main.xml]: no caps in string titles.

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 4]
```diff
@@ -234,6 +234,9 @@
         android:layout_alignParentBottom="true"
         android:layout_alignRight="@+id/relativeLayout2"
         android:layout_marginBottom="@dimen/continue_margin"
-        android:background="@drawable/endingscreen6" />
+        android:layout_marginLeft="45dp"
```
Comment (chhavip) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: no hard coding.

File: PowerUp/app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -55,4 +54,9 @@
     <string name="about_urgency_section_title"><![CDATA[> Why this App is needed]]></string>
     <string name="about_how_app_helps_section_title"><![CDATA[> How does PowerUp help teenagers?]]></string>
     <string name="about_how_app_helps_section_content">The app uses Social and Emotional Learning (SEL) to empower middle school girls to take ownership of their health and sexual activity. It is designed to increase access to reproductive health and self-esteem education by incorporating quality and relevant information surrounding those topics and it helps concretize corresponding concepts through interactive activities. The Content is based on SEL Core Competencies (Self-awareness, Self-management, Social awareness, Relationship skills, Decision making).</string>
+    <string name="New_User"><b>New User</b></string>
+    <string name="Start"><b>Start</b></string>
```
Comment (chhavip) [inline PowerUp/app/src/main/res/values/strings.xml]: No caps in titles and use names lie new_user_text

**Comment by chhavip** [comment]
The work done is good! Some minor formatting changes.

**Comment by mahikaw** [comment]
Please review @chhavip

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 6]
```diff
@@ -134,7 +134,7 @@ protected void onCreate(Bundle savedInstanceState) {
             final TextView tvPoints = (TextView) findViewById(R.id.tvSelectFeaturePoints);
             tvPoints.setText(String.valueOf(getmDbHandler().getPointsClothes(cloth)));
             tv.setText(R.string.cloth);
-            // Left and right buttons allow user to scroll through clothing options
+            // Left and right brown_button_background allow user to scroll through clothing options
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Comment wrongly changed

**Comment by chhavip** [comment]
Not all changes asked for before have been made like of hardcoded dimens and capital starting of strings. Please dont ask for reviews before thoroughly going through the code yourself.

**Comment by mahikaw** [comment]
I'm really sorry @chhavip . I have updated and checked thoroughly now.

File: PowerUp/app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -29,22 +29,28 @@
...
         android:src="@drawable/ic_launcher" />
 
-    <ImageButton
+    <Button
         android:id="@+id/startButtonMain"
         android:layout_width="@dimen/start_button_width"
         android:layout_height="@dimen/start_button_height"
-        android:background="@drawable/buttons" />
+        android:background="@drawable/brown_button_background"
+        android:text="@string/start"
```
Comment (aanchal07) [inline PowerUp/app/src/main/res/layout/activity_main.xml]: No resource found that matches the given name (at 'text' with value '@string/start').

File: PowerUp/app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -29,22 +29,28 @@
...
+        android:textSize="@dimen/button_font"/>
 
-    <ImageButton
+    <Button
         android:id="@+id/aboutButtonMain"
         android:layout_width="@dimen/about_button_width"
         android:layout_height="@dimen/about_button_height"
-        android:background="@drawable/aboutbutton" />
+        android:background="@drawable/brown_button_background"
+        android:text="@string/about"
```
Comment (aanchal07) [inline PowerUp/app/src/main/res/layout/activity_main.xml]: No resource found that matches the given name (at 'text' with value '@string/about').

**Comment by aanchal07** [comment]
There some missing string values. I have added comments please check.

**Comment by mahikaw** [comment]
Updated. 😄

**Comment by chhavip** [comment]
@aanchal07, looks good to me, please look over once too if you find anything missing.

**Comment by aanchal07** [comment]
@chhavip I am still having that same error. Maybe it is because of the emulator. 

image

**Comment by mahikaw** [comment]
@aanchal07 . I looked up the issue, it is most probably because of the emulator setting "largeHeap=true" can solve it. If you want I can update this. :)

**Comment by chhavip** [comment]
@aanchal07 @anubhakushwaha Can you please test on some devices and let us know the results here?

**Comment by anubhakushwaha** [comment]
@chhavip @aanchal07 I tested it on 
* ONE PLUS X (ONE E1003)
* NEXUS 5
And it's showing no errors.

**Comment by aanchal07** [comment]
Alright then I guess @chhavip can merge it. Good work @mahikaw

**Comment by anubhakushwaha** [comment]
@aanchal07 On which device you tested when it gave those errors?

**Comment by aanchal07** [comment]
@anubhakushwaha Genymotion Nexus 5 emulator.

**Comment by mahikaw** [comment]
A gentle reminder. @chhavip

**Comment by anubhakushwaha** [comment]
@aanchal07 Could you just check again because it is not showing errors when I check on various devices and accordingly we can make the progress with this PR.

**Comment by chhavip** [comment]
Looks good to me, will merge if no more problems are reported by tomorrow @aanchal07

**Comment by mahikaw** [comment]
A gentle reminder.

**Comment by chhavip** [comment]
Merging. Good work done @mahikaw

**Pull request merged**